db/data.py

```diff
@@ -460,3 +461,40 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def check_low_level_duplicates(data):
+    """
+        Check if data comes from an audio file already analysed
+        Uses the md5_encoded field
+    Args: data
+        A dictionary containing the low-level data extracted using essentia
```

Comment:
A bit of nitpicking. Most of the docstrings are formatted based on this https://google.github.io/styleguide/pyguide.html#Comments. So that docstring could look like this:
```
Checks if data with the same md5_encoded value has already been submitted.

Args:
    data: Submitted data.
```